### PR TITLE
[READY-TO-MERGE] Add space updated and deleted webhook doc

### PR DESCRIPTION
**Pull request by steveodonovan**
@miguelestrada @jarusso Could I get a review please ? Also aside from the dx web client and this is there anywhere else to be updated ?

Corresponding internal discussion 33252 32787
Corresponding internal tickets 33201, 33199

### CODE REVIEW

File: references/V1_OutboundCallback.md [hunk 1]
```diff
@@ -243,6 +245,43 @@ This event is sent only to webhooks that
 |**time**  <br>*required*|Time and date of member removal, in milliseconds since January 1st, 00:00, 1970 UTC|integer(int64)|
 |**type**  <br>*required*|The event type is `space-members-removed`.|string|
 
+<a name="spaceupdatedbody"></a>
+### SpaceUpdatedBody
+Notifies that a space has been updated.
+
+This event is sent only to webhooks that
+- have been added for the message-updated event
```
Comment (AlanGreene) [inline references/V1_OutboundCallback.md]: `space-updated` ?

File: references/V1_OutboundCallback.md [hunk 2]
```diff
@@ -243,6 +245,43 @@ This event is sent only to webhooks that
...
+
+
+|Name|Description|Schema|
+|---|---|---|
+|**spaceId**  <br>*required*|Id of the space which was updated|string|
+|**userId**  <br>*required*|Id of the user who updated the space.|string|
+|**type**  <br>*required*|The event type is `space-updated`.|string|
+|**time**  <br>*required*|Time and date of the space being updated, in milliseconds since January 1st, 00:00, 1970 UTC|integer(int64)|
+|**title**  <br>*optional*|The new title of the space.|string|
+|**description**  <br>*optional*|The new title of the space.|string|
```
Comment (AlanGreene) [inline references/V1_OutboundCallback.md]: `The new description of the space.`

File: references/V1_OutboundCallback.md [hunk 3]
```diff
@@ -243,6 +245,43 @@ This event is sent only to webhooks that
...
+|**visibility**  <br>*optional*|The new visibility of the space.|string|
+|**allowGuests**  <br>*optional*|The new allow guests setting of the space.|string|
+|**statusValue**  <br>*optional*|The new status value of the space.|string|
+|**spaceProperties**  <br>*optional*|The new space property values of the space.|Map<String, Object>|
+
+<a name="spaceupdatedbody"></a>
```
Comment (AlanGreene) [inline references/V1_OutboundCallback.md]: `spacedeletedbody`

File: references/V1_OutboundCallback.md [hunk 4]
```diff
@@ -243,6 +245,43 @@ This event is sent only to webhooks that
...
+### SpaceDeletedBody
+Notifies that a space has been deleted.
+
+This event is sent only to webhooks that
+- have been added for the message-deleted event
```
Comment (AlanGreene) [inline references/V1_OutboundCallback.md]: `space-deleted`

**Review by AlanGreene — CHANGES_REQUESTED** [review]

**Comment by jarusso** [comment]
@trik-1 looks good - but need you to also make an entry in the What's New md file about this change/update

**Comment by steveodonovan** [comment]
@jarusso could I get another once over please ?

**Review by miguelestrada — APPROVED** [review]

**Comment by steveodonovan** [comment]
@miguelestrada Missed out on one small part added in last commit. Should be ready to merge.

**Pull request merged**